examples/quickstart/jupyter-notebooks/notebooks/03-query/04-UnionOperations.ipynb

```diff
@@ -0,0 +1,644 @@
+    "\n",
+    "```sql\n",
+    "SELECT \"Flights\", \"Shortest\", \"Longest\" from set1\n",
+    "UNION ALL\n",
+    "SELECT \"Frights\", \"Shortest\", \"Lengthiest\" from set2\n",
+    "```\n",
+    "\n",
+    "Run the cell below to see what difference this makes:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "39f9be92-7b2e-417c-b16a-5060b8cd2c30",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sql='''\n",
+    "WITH\n",
+    "set1 AS (\n",
+    "    SELECT\n",
+    "        \"Reporting_Airline\",\n",
+    "        COUNT(*) AS \"Flights\",\n",
+    "        MIN(Distance) AS \"Shortest\",\n",
+    "        MAX(Distance) AS \"Longest\"\n",
+    "    FROM \"On_Time_Reporting_Carrier_On_Time_Performance_(1987_present)_2005_11\"\n",
+    "    WHERE DATE_TRUNC('HOUR', __time) = TIMESTAMP '2005-11-01 11:00:00'\n",
+    "    AND \"Reporting_Airline\" LIKE 'A%'\n",
+    "    GROUP BY 1\n",
+    "  ),\n",
+    "set2 AS (\n",
+    "    SELECT\n",
+    "        \"Reporting_Airline\",\n",
+    "        COUNT(*) AS \"Frights\",\n",
+    "        MAX(Distance) AS \"Lengthiest\",\n",
+    "        MIN(Distance) AS \"Shortest\"\n",
+    "    FROM \"On_Time_Reporting_Carrier_On_Time_Performance_(1987_present)_2005_11\"\n",
+    "    WHERE DATE_TRUNC('HOUR', __time) = TIMESTAMP '2005-11-02 11:00:00'\n",
+    "    AND \"Reporting_Airline\" LIKE 'A%'\n",
+    "    GROUP BY 1\n",
+    "  )\n",
+    "\n",
+    "SELECT \"Flights\", \"Shortest\", \"Longest\" from set1\n",
+    "UNION ALL\n",
+    "SELECT \"Frights\", \"Shortest\", \"Lengthiest\" from set2\n",
+    "'''\n",
+    "\n",
+    "display.sql(sql)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f31b3244-f6bc-4ba6-bb4d-7fb4057d222e",
+   "metadata": {},
+   "source": [
+    "## Working with concatenated result sets\n",
+    "\n",
+    "So far we have seen \"[top level](https://druid.apache.org/docs/26.0.0/querying/sql.html#top-level)\" UNIONs in operation. That is, the `UNION ALL` is being used very simply at the highest level of the query.\n",
+    "\n",
+    "When we want to apply functions to the _entire_ result set like above, such as using it in a sub-query or a `FROM` clause, Druid changes how it plans and executes the query. Instead of a top level `UNION ALL`, it switches to a \"[table level](https://druid.apache.org/docs/26.0.0/querying/sql.html#table-level)\" query.  For example, to put our very first cell results into `__time` order, we might want to use a query that applies `ORDER BY` to a `UNION` of `set1` (San Francisco flights) and `set2` (Atlanta flights), like the following:\n",
+    "\n",
+    "```sql\n",
+    "SELECT \"Origin\",\n",
+    "    \"Tail_Number\",\n",
+    "    \"Flight_Number_Reporting_Airline\"\n",
+    "FROM (\n",
+    "    SELECT * from set1\n",
+    "    UNION ALL\n",
+    "    SELECT * from set2\n",
+    "    )\n",
+    "ORDER BY __time\n",
+    "```\n",
+    "\n",
+    "Run the cell below to see what happens."
```

Comment:
Preface that this query shouldn't work? Something as subtle as, "However, see what happens when Druid tries to run the following query" or more obvious like "However, Druid does not support this query pattern. See what happens:"